gtk2_ardour/po/fr.po

```diff
@@ -11868,7 +11874,6 @@ msgid "OFF"
 msgstr "éteint"
 
 #: monitor_section.cc:238
-#, fuzzy
 msgid "SiP Cut"
 msgstr "Réduction SeP"
```

Comment:
sûr de toi ici @jujudusud ?

Reply:
"SiP" c'est "Solo in place. "SiP Cut" c'est la valeur de coupure des pistes non solotées quand l'écoute se fait en mode Solo.
Donc, je peux proposer "Taux de coupure Sep" et aussi de traduire tous les "SiP" en Sep.
Tu en penses quoi toi ?

Reply:
"Coupure SeP", ça serait bon ?

Reply:
Si on regarde le contexte, je ne sais pas quoi mettre de suffisamment court.
En fait, les pistes non sélectionnées pour le solo sont soit muettes, soit à -20 ou -12 ou -6 dB ou n'importe quelle valeur...
![Screenshot_2022-11-22_23-25-33](https://user-images.githubusercontent.com/12748768/203433846-4b8e95da-70ce-4377-bd25-2f1a824cbaa3.png)

Reply:
Et si tu mets "Coupure SeP", ça fonctionne dans l'IGU ?

Reply:
![Capture d’écran_2022-11-25_17-11-47](https://user-images.githubusercontent.com/12748768/204023648-31f33222-846e-410f-aa1b-1dd17ef31f91.png)

Je pense que ce sera très bien.

Reply:
Ca me paraît très bien, oui :+1: